frontend/src/customizations/volto/components/theme/Footer/Footer.jsx

```diff
@@ -99,6 +99,11 @@ const Footer = ({ intl }) => {
                   <List.Item>
                     <a href="https://ploneconf.org">Conference</a>
                   </List.Item>
+                  <List.Item>
+                    <Link href="/news-and-events/join-the-plone-newsletter">
```

Comment:
@Adityaadpandey Look carefully -- the `Link` component uses `to` instead of `href`